clients/src/main/java/org/apache/kafka/common/requests/OffsetCommitRequest.java

```diff
@@ -351,7 +350,7 @@ public String memberId() {
         return memberId;
     }
 
-    @Deprecated
+    // Note, the retention time is only used in OffsetCommitRequest v2
```

Comment:
Removed the `@Deprecated` annotation since both the Kafka broker and the Kafka client need to support old version of the RPC.